src/refereecore.cpp

```diff
@@ -0,0 +1,25 @@
+#include "refereecore.h"
+
+RefereeCore::RefereeCore() {
+    // Creating world pointer
+    _world = new World();
+}
+
+RefereeCore::~RefereeCore() {
+    // Deleting world module
+    delete _world;
+}
+
+void RefereeCore::start() {
+    // Creating vision pointer and adding it to world with priority 0
+    _vision = new Vision();
+    _world->addEntity(_vision, 0);
+
+    // Starting entities
+    _world->startEntities();
+}
+
+void RefereeCore::stop() {
```

Comment:
The **stop()** method in **RefereeCore** class will make **World** class stop and delete all modules.